inblockio/DataAccounting
# Move modifications from VerifiedWikiImporter to hooks

This allows removing VerifiedWikiImporter, and just use the normal WikiImporter.

WikiImporter not removed in this commit yet. Changes also not tested in any way.

With this commit, both Special:Import and Special:VerifiedImport will cause the
code in the hooks to be executed. This might be undesired. I am currently not
sure what the implications are, perhaps standard import is broken now.

Next steps:

* Test
* Remove VerifiedWikiImporter if we indeed moved all modifications
* Update SpecialVerifiedImport to use WikiImporter
* See if we can reduce duplication in SpecialVerifiedImport
* Move the code out of Hooks class into dedicated services, esp
  the 200 LOC in onImportHandleRevisionXMLTag
